input/include/NovelRT/Input/InputAction.hpp

```diff
@@ -16,6 +16,17 @@ namespace NovelRT::Input
         NovelKey pairedKey;
         KeyState state = KeyState::Idle;
     };
+
+    inline bool operator==(InputAction const& lhs, InputAction const& rhs) noexcept
+    {
+        return lhs.state == rhs.state && (lhs.pairedKey == rhs.pairedKey) && lhs.actionName == rhs.actionName;
+    }
+
+    inline bool operator!=(const InputAction& lhs, const InputAction& rhs) noexcept
+    {
+        return (lhs.state != rhs.state) || (lhs.pairedKey != rhs.pairedKey) || (lhs.actionName != rhs.actionName);
+    }
+
```

Comment:
Is there a reason we chose non-member operators? Just curious

Reply:
Found that these may not be needed anymore - removing them.